src/components/AppLayout/Notifications/Notification.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { TNotificationObject } from "./types";
+
+const Notification = ({
+    icon,
+    title,
+    message,
+    buttonAction,
+    actionText,
+}: TNotificationObject) => {
+    return (
+        <div className="notification">
```

Comment:
```suggestion
        <div className="deriv-notification">
```

This is the way we name our classes in UI

Reply:
having brand name in project is not recommended, what if we want change the brand name?